cpp/ql/src/semmle/code/cpp/security/ExternalAPIs.qll

```diff
@@ -0,0 +1,50 @@
+/**
+ * Definitions for reasoning about untrusted data used in APIs defined outside the
+ * database.
+ */
```

Comment:
This whole qll file seems to me like it's highly specific to the queries that are using it. It doesn't look like it presents an API we want to maintain for out-of-tree code to use. What do you think?

Reply:
I think that's correct. I suppose it can be used for custom exploration queries (i.e., a version of `UntrustedDataToExternalAPI` that applies _some_ filtering)?

If we want to avoid out-of-tree code uses, would it be more appropriate to move the `ExternalAPIs` files into the `Security/CWE/CWE-020` folder and use them from there?

Reply:
> If we want to avoid out-of-tree code uses, would it be more appropriate to move the `ExternalAPIs` files into the `Security/CWE/CWE-020` folder and use them from there?

Yes, I think so. Out-of-tree users can always copy-paste the files if they want.

Reply:
Done in 38feeeb92b09e0e3df899e37cface8f67382ba3d.